scripts/service-update-scripts/kubernetes-merge-branch.sh

```diff
@@ -30,7 +32,7 @@ git checkout dcos-kubernetes/$branch docs/package
 # remove any user specified directories 
 if [ -n "$skip" ]; then 
   echo "Skipping $skip"
-  for d in $(echo $skip | sedi "s/,/ /g"); do rm -rf docs/package/$d; done
+  for d in $(echo $skip | sed "s/,/ /g"); do rm -rf docs/package/$d; done
```

Comment:
Will this work as expected in macOS?

Reply:
Yes, `sedi`( `-i` ) is only needed when working with files, this is using `stdout` so `-i` actually breaks it on MacOS.